.github/release.yml

```
@@ -0,0 +1,22 @@
+# .github/release.yml
+
+changelog:
+  exclude:
+    labels:
+      - Infrastructure
+  categories:
+    - title: Exciting New Features 🎉
+      labels:
+        - Feature Request
+    - title: Bug fixes 🛠️
+      labels:
+        - bug
+    - title: C# on LSP (Omnisharp Parity)
```

Does this need to be wrapped in `''` as '#' means comment in YAML?
```suggestion
    - title: 'C# on LSP (Omnisharp Parity)'
```